src/import_export/univcsvdialog.cpp

```diff
@@ -647,10 +668,10 @@ const wxString mmUnivCSVDialog::getCSVFieldName(int index) const
 {
     std::map<int, wxString>::const_iterator it = CSVFieldName_.find(index);
     if (it != CSVFieldName_.end()) {
-        return wxGetTranslation(it->second);
+        return (it->second);
     }
 
-    return _("Unknown");
+    return wxTRANSLATE("Unknown");
```

Comment:
why?

Reply:
it's OK
https://github.com/vomikan/moneymanagerex/blob/%231122/src/import_export/univcsvdialog.cpp#L1154